Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:18.04
+
+COPY . /usr/src/
```

Comment:
This step is better after apt-get and install rust, so layers can be can cached

Reply:
Changed. Good catch.